AppDB/appscale/datastore/fdb/sdk.py

```diff
@@ -0,0 +1,241 @@
+import logging
```

Comment:
I feel a little ambivalent about changing the style for code copied from the SDK since it would make it harder to diff if someone is trying to see if there are any changes from how the SDK does it.

I'm hoping to remove this module very soon, though, so hopefully the inconsistent style won't be an issue for much longer.